src/sentry/integrations/bitbucket/issues.py

```diff
@@ -29,7 +33,7 @@ def get_repo_choices(self, **kwargs):
         except ApiError:
             repo_choices = []
         else:
-            repo_choices = [(repo['uuid'], repo['full_name']) for repo in repos['values']]
+            repo_choices = [(repo['full_name'], repo['full_name']) for repo in repos['values']]
```

Comment:
Is `full_name` guaranteed to be unique?

Reply:
It's unique but could change: https://developer.atlassian.com/bitbucket/api/2/reference/meta/uri-uuid#repo-obj

Reply:
to use just the repo UUID we'd need to pass in an empty team or username